pytorch/training/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -295,4 +295,4 @@ RUN HOME_DIR=/root \
  && rm -rf /tmp/tmp*
 
 ENTRYPOINT ["bash", "-m", "start_with_right_hostname.sh"]
-CMD ["/bin/bash"]
+CMD ["/bin/bash"]
```

Comment:
Is there a specific reason for making these changes?

Reply:
I haven't made any change, I just reverted it. Not sure why its appearing as a change.

Reply:
Sure, lets rectify it